eq_translations/extractable_strings.py

```diff
@@ -6,6 +6,18 @@
     {"json_path": "$.submission.guidance", "description": "Submission guidance",},
     {"json_path": "$.submission.title", "description": "Submission title"},
     {"json_path": "$.submission.warning", "description": "Submission warning"},
+    {
+        "json_path": "$.post_submission.guidance.contents[*].title",
+        "description": "Post submission guidance heading",
+    },
+    {
+        "json_path": "$.post_submission.guidance.contents[*].description",
+        "description": "Post submission guidance description",
+    },
+    {
+        "json_path": "$.post_submission.guidance.contents[*].list[*]",
+        "description": "Post submission guidance list item",
+    },
```

Comment:
Any reason to not add `"additional_context": ["ListHeading", "ListDescription"],` for the list items?. We add these for the other lists.

Reply:
Good spot, fixed in https://github.com/ONSdigital/eq-translations/pull/76/commits/c7b5741df2b53946aac4110d4849ea92232be861